model/src/pyrenew/deterministic/deterministic.py

```diff
@@ -75,16 +81,27 @@ def sample(
         Parameters
         ----------
         record : bool, optional
-            Whether to record the value of the deterministic RandomVariable. Defaults to True.
+            Whether to record the value of the deterministic
```

Comment:
Why does this line change? Can we check this type of formatting with precommit?

Reply:
That was me disliking the long line. But yes, we should be able to set a maximum docstring line length etc